addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -130,7 +130,7 @@ def _make_request(self, url, params=False, auth_type: Literal['hmac', 'asymmetri
             if error_code == 'refresh_token_expired':
                 self._renew_token()
                 self.env.cr.commit()  # We do not want to lose it if in the _make_request below something goes wrong
-                return self._make_request(url, params, request_timeout)
+                return self._make_request(url, params, 'hmac', request_timeout)
```

Comment:
```suggestion
                return self._make_request(url, params=params, auth_type='hmac', request_timeout=request_timeout)
```
Always a bad idea to call named parameters in a positional way imo.
You can even remove the `auth_type` then if you prefer.